component/portal/src/main/java/org/exoplatform/portal/mop/rest/NavigationRest.java

```diff
@@ -459,6 +459,13 @@ public List<Map<String, Object>> getPageAccessPermissions() {
     public void setPageAccessPermissions(List<Map<String, Object>> pageAccessPermissions) {
       this.pageAccessPermissions = pageAccessPermissions;
     }
+    public String getTarget() {
+      return userNode.getTarget();
+    }
+
+    public String getDescription() {
```

Comment:
description is not yet added as field in frontend, so IMO to be defined here later